pitest-maven/src/main/java/org/pitest/maven/report/AbstractPitAggregationReportMojo.java

```diff
@@ -63,6 +63,7 @@ protected void executeReport(final Locale locale)
           .build();
 
       reportAggregator.aggregateReport();
+      reportAggregator.aggregateXmlReport(false);
```

Comment:
Thinking about the default login in the Gradle plugin. Should the XML report generated by default? Only if XML report is configured for the PIT "single" report? Or there should be a separate property for the aggregation mojo? WDYT?